commands/genpoke.js

```diff
@@ -6,25 +6,31 @@ const logger = require('../logs/logger.js');
 const CMD_TEMPLATE = '+genpoke [SPECIES] [LEVEL (1-20)] [NICKNAME - no spaces or special characters] [HIDDEN' +
 	' ABILITY % (as a number, 0-100)]';
 //help message
+const HELP_MESSAGE = '\n`+genpoke [species] [level (1-20)] [nickname] [Form Name] [hidden ability % (as a number, 0-100)]\n' +
```

Comment:
Update: remove actual "+genpoke" line (covered in CMD_TEMPLATE); remove "fields to add later"

Reply:
done